### PR TITLE
fix: currency selection issues

### PR DESCRIPTION
**Pull request by rodrigopavezi**
## Changes

refactor: Clean up imports and improve currency handling in invoice forms

- Reformatted import statements for better readability.
- Enhanced currency filtering logic to prevent duplicates and handle specific cases for ISO4217 currencies.
- Updated the invoice currency dropdown display to simplify the output.
- Added error handling in balance formatting to improve robustness.

These changes aim to streamline the invoice creation process and enhance user experience.

https://github.com/user-attachments/assets/f72895e3-7b1c-44f0-8e62-1d2adcfababf

## Summary by CodeRabbit

- **New Features**
	- Enhanced validation for email addresses and Ethereum addresses in the invoice form.
	- Improved handling of invoice items with dynamic input management.

- **Bug Fixes**
	- Added error handling in balance formatting to prevent runtime errors.
	- Enhanced logging for missing parameters in balance checking.

- **Chores**
	- Updated version numbers for `@requestnetwork/create-invoice-form` and `@requestnetwork/invoice-dashboard` packages.

### CODE REVIEW

File: packages/invoice-dashboard/src/lib/dashboard/invoice-view.svelte [hunk 1]
```diff
@@ -480,9 +480,14 @@
...
-      ? value.toString()
-      : value.toFixed(maxDecimals);
+    try {
+      return Number.isInteger(value)
+        ? value.toString()
+        : value.toFixed(maxDecimals);
+    } catch (error) {
+      console.error("Error formatting balance:", error);
+      return "-";
+    }
```
[the remark this answers is not in the document]
Comment (rodrigopavezi) [reply packages/invoice-dashboard/src/lib/dashboard/invoice-view.svelte:490]: I will leave like that for now

**Comment by MantisClone** [comment]
When you clear the invoice currency without making a new selection will it still clear the other two fields?

**Comment by rodrigopavezi** [comment]
It will not until you selected a different one. I think the Dropdown only triggers the handler when selecting

**Comment by MantisClone** [comment]
Okay

**Review by MantisClone — APPROVED** [review]
I didn't look at the code.

**Pull request merged**